src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
+      background: var(--sapPageFooter_Background);
+      height: 3rem;
+
+      @include fd-ellipsis();
+
+      &--compact {
+        size: 1.625rem;
+        font-size: var(--sapFontSize);
+      }
+
+      .#{$block}__title {
+        -webkit-box-flex: 1;
+        flex: auto;
+        width: auto;
+        min-width: 3rem;
+        display: block;
+        text-align: center;
+        font-weight: normal;
+        color: var(--sapButton_TextColor);
+        font-size: var(--sapFontLargeSize);
+        text-shadow: none;
+
+        @include fd-ellipsis();
+        @include fd-rtl();
+      }
+    }
+
+    &--display {
+      display: -webkit-box;
+      display: flex;
+      -webkit-box-flex: 1;
+      flex: 1 1 auto;
+      max-width: 100%;
+
+      @include fd-ellipsis();
+
+      &--action {
+        cursor: pointer;
+      }
+
+      .#{$block}__title {
+        -webkit-box-flex: 1;
+        flex: auto;
+        min-width: 3rem;
+        display: block;
+        font-size: var(--sapFontLargeSize);
+        -webkit-box-align: center;
+        align-items: center;
+        color: var(--sapList_TextColor);
+        font-weight: normal;
+
+        @include fd-ellipsis();
+
+        @include fd-rtl() {
+          text-align: right;
+        }
+      }
+
+      .#{$block}__secondary {
+        -webkit-box-flex: 1;
+        flex: auto;
+        min-width: 3rem;
+        font-size: 0.875rem;
+        margin-left: 1rem;
+        text-align: right;
+        color: var(--sapContent_LabelColor);
+
+        @include fd-ellipsis();
+
+        @include fd-rtl() {
+          text-align: left;
+        }
+      }
+
+      .#{$block}__icon {
+        font-size: 0.75rem;
+        width: 2.75rem;
+        color: var(--sapContent_NonInteractiveIconColor);
+        -webkit-box-flex: 0;
+        flex: none;
+        box-sizing: border-box;
+        cursor: pointer;
+        text-align: center;
+        display: inline-block;
+        vertical-align: baseline;
+
+        &:active {
+          color: var(--sapList_Active_TextColor);
+        }
```

Comment:
Please check if you really need all this code, some if it might have already been defined in the Standard List

Reply:
Right now list.scss has

```
 &__icon {
    @include fd-reset();

    color: var(--sapList_TextColor);
    flex: 0 0 auto;
    width: 2.75rem;
    display: flex;
    justify-content: center;
    align-items: center;
    font-size: var(--sapFontHeader4Size);
```
As per my understanding once #852 issue is fixed then these values will be updated as per Fiori specfication as below.
<img width="309" alt="Screen Shot 2020-04-17 at 1 01 10 PM" src="https://user-images.githubusercontent.com/53487468/79543742-92dde500-80ab-11ea-98c0-d1fcebd50bf7.png">



So as of now, I removed
```
width: 2.75rem;
-webkit-box-flex: 0;
```
Other values will update once the #852 fix is merged.